pillar/components/upgrade.sls

```diff
@@ -0,0 +1,12 @@
+upgrade:
+  sw_list:
```

Comment:
Should this be renamed to cortx_components?

Reply:
@andkononykhin2  
Can we change this to cortx_components? please suggest

Reply:
I would say 3rd parties may appear here someday as well and we might use that list as a flat sequence that presented dependency hierarchy of the whole CORTX sw stack (not only cortx). Let's keep that as-is for now and re-think later if needed.

Reply:
@andkononykhin2 
Are we good to merge this PR?